ansible/roles/kubeadm/tasks/redhat.yaml

```diff
@@ -14,9 +14,34 @@
       not 'kubeadm-' + package_versions.kubernetes_rpm in exportedversionlocklist.stdout
       )"
 
+
+- block:
+  - name: copy cri-tools rpm
+    copy:
+      src: "/opt/{{ 'cri-tools-' + critools_rpm }}{{ '-fips' if fips.enabled else '' }}.rpm"
+      dest: "/opt/{{ 'cri-tools-' + critools_rpm }}.rpm"
```

Comment:
should we keep the `-fips` suffix for fips rpms? it would be easy to debug in that case. There is no other way to identify if the rpm is fips enabled.

Reply:
The same goes for all other rpms below.

Reply:
I don’t think so, when we download these files from our fips repository they don’t have a suffix. I’d prefer to stay consistent with what we have there.